src/unittests/f32_interval.spec.ts

```diff
@@ -3563,3 +3564,200 @@ g.test('transposeInterval')
       )}]'. Expected '[${JSON.stringify(expected)}]'`
     );
   });
+
+interface MatrixPairCase {
+  input: [number[][], number[][]];
+  expected: IntervalBounds[][] | number[][];
+}
+
+g.test('additionMatrixInterval')
+  .paramsSubcasesOnly<MatrixPairCase>([
+    // Only testing that different shapes of matrices are handled correctly, to
+    // reduce test duplication. This function uses AdditionIntervalOp for
+    // calculating intervals, so depending on additionInterval's testing to
+    // make sure that things like subnormals are handled as expected.
```

Comment:
Just checking, but this comment about `AdditionIntervalOp` is mostly informative, right, since none of the test cases would actually exercise subnormal handling?

Reply:
That is correct, I have cleaned up the comment a bit to hopefully make it clearer.